system/Database/Database.php

```diff
@@ -54,6 +56,8 @@ public function load(array $params = [], string $alias = '')
             throw new InvalidArgumentException('You have not selected a database type to connect to.');
         }
 
+        assert($this->checkDbExtension($params['DBDriver']));
```

Comment:
Do we need the assert here if the method throws by itself?

Reply:
My intention is not to slow down CI4 at all by executing the check every time in production environment.
When `zend.assertions` is `-1`, PHP does not generate code for `assert()`.

First, this check (error message) is only for PHP beginners.
And this error often happens on development environments.
On shared hosting servers, they probably know what DBMS is supported, and PHP extensions for supported DBs are installed.